sources/package.json

```diff
@@ -6,11 +6,16 @@
 	"scripts": {
 		"doc": "jsdoc -c  jsdoc.json -r",
 		"prettier": "npx prettier --write .",
-		"eslint": "npx eslint .",
-		"start": "node index.js",
-		"start:dev": "nodemon index.js",
-		"test": "NODE_ENV=test jest --runInBand --coverage",
-		"test:watch": "NODE_ENV=test jest --runInBand --watch"
+		"prettier:check": "npx prettier --check .",
+		"eslint": "npx eslint",
+		"eslint:fix": "npx eslint --fix",
+		"start": "npx pm2 start ecosystem.config.js --only dev --env development",
+		"stop": "npx pm2 stop ecosystem.config.js --only dev --env development",
+		"delete": "npx pm2 delete ecosystem.config.js --only dev",
+		"restart": "npx pm2 restart ecosystem.config.js --only dev --env development",
+		"start:prod": "pm2-runtime start ecosystem.config.js --only prod --env production",
+		"test": "cross-env NODE_ENV=test jest --runInBand --coverage",
+		"test:watch": "cross-env NODE_ENV=test --runInBand --watch"
```

Comment:
@DesignrKnight Can we have a status script here? Even pm2 status wont work

Reply:
Done in https://github.com/dscnitrourkela/project-dates/pull/82/commits/cb976323395b63c7b0ee4e57241c0aad357bc68a